modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -510,7 +510,7 @@ const register = (editor: Editor): void => {
 
   registerOption('iframe_aria_text', {
     processor: 'string',
-    default: 'Rich Text Area. Press ALT-0 for help.'
+    default: editor.hasPlugin('help') ? 'Rich Text Area. Press ALT-0 for help.' : 'Rich Text Area'
```

Comment:
Instead of writing two different strings, I would do this instead, to avoid introducing typos:

```suggestion
    default: 'Rich Text Area.'.concat(editor.hasPlugin('help') ? ' Press ALT-0 for help.' : '')   
```